### DISCUSSION TITLE
Instance getting repeatedly re-scheduled

### DISCUSSION THREAD

**Discussion by ericoporto** [cirruslabs/cirrus-ci-docs#1225] [Q&A] [answered]
We build for Android and because we use ninja for c++ and gradle we ask for slightly more resources (CPU+RAM) for this task specifically. I mention in case it's related.

Our task is getting repeatedly re-scheduled: https://cirrus-ci.com/task/5463134035509248

Just wanted to check if the issue is on our side somehow - we didn't change anything from what we can tell, and it used to work fine.

https://github.com/adventuregamestudio/ags/blob/master/Android/gradle/buildCacheSettings.gradle

The Gradle script uses the cache from CirrusCI not sure if relevant.

**Comment by ericoporto** [comment]
It seems now it keeps returning a pod failed error after some time

https://cirrus-ci.com/task/5487899790016512

Any idea what this means?

**Comment by ericoporto** [comment]
I see that the log appears like the gradle daemon doesn't run... But I don't know why...

Any chance of upgrading the sdk for 34 ??

https://github.com/cirruslabs/docker-images-android/issues/58

We haven't changed anything there gradle wise but maybe changing it will work...

**Comment by edigaryev** [comment, marked as answer]
Hello Érico,

Thanks for reporting this! The cause seems to be:

```
3:54:51 AM Pod ephemeral local storage usage exceeds the total limit of containers 1Gi.
3:54:51 AM Stopping container main
```

We've just deployed a fix that should substantially increase the ephemeral storage sizes for pods.

**Reply by ericoporto** [reply]
@edigaryev I confirm the fix worked on my side! Thank you so much!!!!
